shared/constants/metametrics.ts

```diff
@@ -455,6 +455,10 @@ export enum MetaMetricsUserTrait {
    */
   MmiIsCustodian = 'mmi_is_custodian',
   ///: END:ONLY_INCLUDE_IN
+  /**
+   * Identified when the security provider feature is enabled.
+   */
+  SecurityAlertsEnabled = 'security_alerts_enabled',
```

Comment:
This is not being used anywhere.. is there a reason to add it?

Reply:
should be renamed to SecurityAlertBlockaidEnabled. :(